cEP-0003.md

```diff
@@ -0,0 +1,96 @@
+Teams in coala
+==============
+
+|Metadata|                                                                    |
+|--------|--------------------------------------------------------------------|
+|cEP     |0003                                                                |
+|Version |0.1                                                                 |
+|Title   |Teams in coala                                           |
+|Authors |Maximilian Scholz <m0hawk@gmx.de>                                   |
+|Status  |proposed                                                            |
+|Type    |Process                                                             |
+
+Abstract
+--------
+
+This is a proposal on how to decentralize the management of coala by
+introducing teams.
+
+Structure
+---------
+
+A team is a self organizing group that is responsible for a certain aspect of
+coala. Code reviews should be done internally or in cooperation with the review
+team.
+
+
+Aspects
+-------
+
+- **core**:
+Developing and maintaining the coala core repository.
+In Addition everything that does not fit into another team's domain, will
+be assigned to the core team.
+
+- **bears**:
+Developing and maintaining the coala-bears repository. Wrap more existing
+tools, help with writing new bears and update existing ones.
+
+- **research**:
+Work on new architecture for coala parts. Help with thesis writing and
+implementing algorithms for the science community.
+
+- **docs**:
+Improve the existing documentation, wiki, tutorials and guides. Introduce new
+ones as needed.
+
+- **plugins**:
+Develop and maintain the coala plugins for multiple text editors and IDEs.
+
+- **engagement**:
+Create all promotional material like fliers, blog posts,
+homepage content and such. Also manage conference attendance and feedback
+handling.
+
+- **review**:
+If there is a big pr or a very complicated one this team is meant to come from
+outside the other teams to help with the review process. Also in general
+this team should train the rest of the org how to do good reviews.
+
+- **Internships**:
+Temporary admin team for things like the Google Summer of Code that don't need
+to be done the whole year.
+
+Roles
+-----
+
+A coala team holds the consists of the following roles.
+
+### Team Leader
+
+The team leader is the main organizer of the team. His job is not to do the
+teams work, but to make sure that everyone knows their tasks. The Team Leader
+is also a Team Member.
+At any time there may be no more than two Team Leaders from the same company or
+organization.
```

Comment:
I think the no more than 2 per company came from you. And I added the org thing because there might be things not considered a company but still want to influence coala.